.changeset/weak-kings-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
+---
+
+Fix redirecting to wrong workflow bug.
```

Comment:
One changeset file should be enough

Reply:
This PR contains fixes for 2 tickets, so I think it's better to specify.